packages/devextreme/js/__internal/scheduler/header/m_date_navigator.ts

```diff
@@ -143,3 +149,14 @@ const isNextButtonDisabled = (header) => {
   const nextDate = header._getNextDate(1, caption.startDate);
   return nextDate > max;
 };
+
+const getCurrentViewText = (header) => {
+  const { currentView } = header;
+  return typeof currentView === 'string'
+    ? currentView
```

Comment:
It is better to avoid ternary operator:
if(typeof header.currentView === 'string') {
    return header.currentView;
}

Reply:
What types in the header.currentView are possible?

Reply:
I'm not sure, but this is for qUnit failing tests. There are several tests that currentView is not a string and returns undefined..